modules/systemd/README.md

```diff
@@ -11,4 +11,23 @@ Those units are then copied into these folders (depending on unit base):
 `/usr/lib/systemd/system`  
 `/usr/lib/systemd/user`
 
-Supported management operations are [enabling](https://www.freedesktop.org/software/systemd/man/latest/systemctl.html#enable%20UNIT%E2%80%A6), [disabling](https://www.freedesktop.org/software/systemd/man/latest/systemctl.html#disable%20UNIT%E2%80%A6), [masking](https://www.freedesktop.org/software/systemd/man/latest/systemctl.html#mask%20UNIT%E2%80%A6%E2%80%A6) and [unmasking](https://www.freedesktop.org/software/systemd/man/latest/systemctl.html#unmask%20UNIT%E2%80%A6).
+Specific systemd config can be included in these directories, depending on certain usecases:  
+`config/systemd/system.conf.d`  
+`config/systemd/user.conf.d`  
+`config/systemd/zram-generator.conf.d`
+
+Which are then copied into these folders:  
+`/usr/lib/systemd/system.conf.d`  
+`/usr/lib/systemd/user.conf.d`  
+`/usr/lib/systemd/zram-generator.conf.d`
+
+Supported systemd unit management operations are:  
+ [enabling](https://www.freedesktop.org/software/systemd/man/latest/systemctl.html#enable%20UNIT%E2%80%A6)  
+[disabling](https://www.freedesktop.org/software/systemd/man/latest/systemctl.html#disable%20UNIT%E2%80%A6)  
+[masking](https://www.freedesktop.org/software/systemd/man/latest/systemctl.html#mask%20UNIT%E2%80%A6%E2%80%A6  
+[unmasking](https://www.freedesktop.org/software/systemd/man/latest/systemctl.html#unmask%20UNIT%E2%80%A6)
+
+Supported systemd configs are:  
+[system.conf.d & user.conf.d](https://www.freedesktop.org/software/systemd/man/latest/systemd-system.conf.html)  
```

Comment:
The reference pages linked are a bit shallow on the details on the use cases for these files. Maybe add a short description of the file and a short example of what they could be used for?